### DISCUSSION TITLE
Commercial use

### DISCUSSION THREAD

**Discussion by ugintl** [YetiForceCompany/YetiForceCRM#15309] [General] [answered]
I want to use it for a client. What are the conditions?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/blob/95e4d31b42933d5c76e41c155fc549f663ba1bf3/LICENSE#L1L15

**Comment by ugintl** [comment]
I have installed yeti, but why is it asking me to register the company? What if I do not want to register?

**Reply by rskrzypczak** [reply]
You have seen the licenses, and as to the legal consequences - ask your lawyer.
Any non-standard activities that you would like to carry out should be sent directly to the manufacturer: https://yetiforce.com/en/company/contact
.. just ask specific questions, and not like - _What if I do not want to .._ :)
